optaplanner-benchmarks/pom.xml

```diff
@@ -31,12 +32,12 @@
       <dependency>
         <groupId>org.kie</groupId>
         <artifactId>optaplanner-perf-framework</artifactId>
-        <version>${project.version}</version>
+        <version>${version.org.optaplanner}</version>
       </dependency>
       <dependency>
         <groupId>org.kie</groupId>
         <artifactId>optaplanner-perf-benchmark</artifactId>
-        <version>${project.version}</version>
+        <version>${version.org.optaplanner}</version>
```

Comment:
If I am not mistaken, this should be kept at `${project.version}` because these modules are not productized. It was also the case before the last PR.

Reply:
You're absolutely correct, forgive my stupidity :) I think I used the ReplaceAll feature :1st_place_medal: